src/django/api/templates/mail/claim_facility_submitted_body.txt

```diff
@@ -1,28 +1,15 @@
 {% block content %}
-Hi,
+Hello,
```

Comment:
There are some inconsistencies in the text/data we show the user in the email between the edited HTML file and this TXT file. Is this expected?
Edited HTML file is in this PR -> https://github.com/opensupplyhub/open-supply-hub/pull/249